src/renderer_gl.cpp

```diff
@@ -7101,7 +7101,7 @@ namespace bgfx { namespace gl
 					{
 						const TextureGL& texture = s_renderGL->m_textures[at.handle.idx];
 
-						if (0 != texture.m_id)
+						if (0 != texture.m_rbo && 0 != texture.m_id)
```

Comment:
Don't try to resolve multisample textures too, ensure it also has an rbo!
(If _one_ attachment required resolve, this would accidentally try and resolve the multisample texture too, causing an unhappy framebuffer)